### PR TITLE
Synchronize with format of ArcaeaUnlimitedAPI (until 2022.05.01

### PR DESCRIPTION
**Pull request by linwenxuan04**
The json format of bot portal is adjusted to enable the developer to adapt the bot for server easily

### CODE REVIEW

File: Team123it.Arcaea.MarveCube/Processors/Front/Bot.cs [hunk 1]
```diff
@@ -78,27 +81,34 @@ public static JObject QueryPlayerBestScore(string user,string songid,SongDifficu
...
 						{
 							cmd.Parameters.Clear();
-							cmd.CommandText = $"SELECT sid FROM fixed_songs WHERE sid LIKE concat('%',?songid,'%') OR name_en LIKE concat('%',?name_en,'%') OR name_jp LIKE concat('%',?name_ja,'%');";
+							cmd.CommandText =
+								$"SELECT sid FROM fixed_songs WHERE sid LIKE concat('%',?songid,'%') OR name_en LIKE concat('%',?name_en,'%') OR name_jp LIKE concat('%',?name_ja,'%');";
 							cmd.Parameters.Add(new MySqlParameter("?songid", songid));
 							cmd.Parameters.Add(new MySqlParameter("?name_en", songid));
 							cmd.Parameters.Add(new MySqlParameter("?name_ja", songid));
-							sid = (string)cmd.ExecuteScalar();
+							sid = (string) cmd.ExecuteScalar();
```
Comment (Misaka12456) [inline Team123it.Arcaea.MarveCube/Processors/Front/Bot.cs:89]: Unnecessary space

File: Team123it.Arcaea.MarveCube/Processors/Front/Bot.cs [hunk 2]
```diff
@@ -362,58 +396,74 @@ public static JObject PlayerInfo(string userNameOrCode)
...
+						cmd.CommandText =
+							$"SELECT COUNT(song_id),song_id,difficulty,score,shiny_perfect_count,perfect_count,near_count,miss_count,health,modifier,time_played,clear_type,rating FROM users WHERE user_id={userid};";
 						rd = cmd.ExecuteReader();
 						rd.Read();
 						JObject recentScoreJObj;
 						if (rd.GetInt32(0) == 1) //如果存在最近成绩
 						{
-							var recentScore = new SingleScore(rd.GetString(1), (SongDifficulty)rd.GetInt32(2),
-								(uint)rd.GetInt32(3), (ClearType)rd.GetInt32(11), (uint)rd.GetInt32(4), (uint)rd.GetInt32(5), (uint)rd.GetInt32(6), (uint)rd.GetInt32(7), (ulong)rd.GetInt64(10) * 1000);
+							var recentScore = new SingleScore(rd.GetString(1), (SongDifficulty) rd.GetInt32(2),
```
Comment (Misaka12456) [inline Team123it.Arcaea.MarveCube/Processors/Front/Bot.cs:424]: Unnecessary spaces

File: Team123it.Arcaea.MarveCube/Processors/Front/Bot.cs [hunk 3]
```diff
@@ -362,58 +396,74 @@ public static JObject PlayerInfo(string userNameOrCode)
...
+								{"score", recentScore.Score!.Value},
+								{"shiny_perfect_count", recentScore.BigPureCount},
+								{"perfect_count", recentScore.PureCount},
+								{"near_count", recentScore.FarCount},
+								{"miss_count", recentScore.LostCount},
+								{"best_clear_type", (int) bestClearType},
+								{"clear_type", (int) recentScore.ClearType},
+								{"health", health},
+								{
+									"time_played",
```
Comment (Misaka12456) [inline Team123it.Arcaea.MarveCube/Processors/Front/Bot.cs:454]: Unnecessary new line

File: Team123it.Arcaea.MarveCube/Processors/Front/Bot.cs [hunk 4]
```diff
@@ -362,58 +396,74 @@ public static JObject PlayerInfo(string userNameOrCode)
...
+									(long) Math.Floor((recentScore.PlayDate!.Value - new DateTime(1970, 1, 1))
```
Comment (Misaka12456) [inline Team123it.Arcaea.MarveCube/Processors/Front/Bot.cs:455]: Unnecessary spaces before type cast

File: Team123it.Arcaea.MarveCube/Processors/Front/Bot.cs (hunk 3 above)
Comment (linwenxuan04) [reply Team123it.Arcaea.MarveCube/Processors/Front/Bot.cs:454]: It is ReSharper's fault, not mine (

File: Team123it.Arcaea.MarveCube/Processors/Front/Bot.cs [hunk 5]
```diff
@@ -548,34 +564,34 @@ public static JObject SongInfo(string songid)
...
+						
+						diff_pst.Add("jacketOverride", false);
+
+						diff_pst.Add("realrating",rd.GetInt32("rating_pst"));
 					}
 					else
 					{
 						diff_pst = new JObject()
 						{
-							{ "ratingClass", 0 },
```
Comment (Misaka12456) [inline Team123it.Arcaea.MarveCube/Processors/Front/Bot.cs:562]: Why delete these spaces?

**Review by Misaka12456 — CHANGES_REQUESTED** [review]
*plz see the comments and then modify it before merge again thx*

**Review by Misaka12456 — APPROVED** [review]
Okay I will change it after merge

**Pull request merged**